core/server/data/import/utils.js

```diff
@@ -136,11 +134,20 @@ utils = {
         return tableData;
     },
 
-    preProcessRolesUsers: function preProcessRolesUsers(tableData) {
+    preProcessRolesUsers: function preProcessRolesUsers(tableData, owner) {
         _.each(tableData.roles_users, function (roleUser) {
             var user = _.find(tableData.users, function (user) {
                 return user.id === parseInt(roleUser.user_id, 10);
             });
+
+            // Check for owner users that do not match current owner and change role
+            // to administrator
+
+            if (roleUser.role_id === 4 && user && user.email && user.email !== owner.email) {
+                roleUser.role_id = 1;
+                user.roles = [1];
+            }
```

Comment:
Although I'm pretty sure that on 99.9% of blogs, the role ids are 1 for admin and 4 for owner, there is no guarantee that this is the case.

For this reason, we have to jump through extra steps to match the role id to a role and then look for the name, as is done for models here: https://github.com/cobbspur/Ghost/blob/owner/core/server/data/import/data-importer.js#L23

For the import here, the role ids should be mapped to the roles listed in the imported JSON, but if they are missing I think it is safe to fall back to the assumption that 1 === admin, and 4 === owner.

In the future, for smarts, it might be useful to define a shorthand for this such that you could define something like: 

```
"users": [{
   id: 3,
   name: "A User",
   email: "user@example.com",
   roles: ["Owner"]
}]
```
(this is from the bare-minimum user map as described in the JSON format doc: https://github.com/TryGhost/Ghost/wiki/import-format#linking-objects-to-users)

This would allow the quick-creation of a user map where each user has a certain role. It is certainly a different thing to this PR and just an idea I wanted to write down whilst I was thinking of it :) You'll know best whether this is a good idea :wink: